bin/cli.js

```diff
@@ -1,8 +1,18 @@
 #!/usr/bin/env node
 const cli = require('cac')()
 const chalk = require('chalk')
+const MaidError = require('../lib/MaidError')
 
 cli.command('*', 'Run a task in current working directory', (input, flags) => {
+  if (flags.updateScripts && input[0]) {
+    throw new MaidError('Cannot run task and update scripts')
```

Comment:
why disallow this 🤔

Reply:
in the updateScripts method, I'm passing all the arguments through transparently. If I allowed this I'd have to find the input and remove it before writing to the package.json. 

So... laziness. I can remove it. Point might be moot if this turns into its own command though.